.github/dependabot.yml

```
@@ -1,14 +1,40 @@
 # https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
 version: 2
 updates:
+
+  # Enable version updates for python
+  - package-ecosystem: "pip"
+    # Look for a `requirements` in the `root` directory
+    directory: "/requirements"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
+    # Labels on pull requests for version updates only
+    labels:
+      - "ci"
```

Needs to match with the label we currently have:
```suggestion
      - "ci/cd"
```
(alternatively, we can rename the GitHub label from `ci/cd` to `ci`)